server/util/consistent_hash/consistent_hash.go

```diff
@@ -33,22 +33,29 @@ var (
 )
 
 type ConsistentHash struct {
-	ring        map[int]uint8
-	keys        []int
-	items       []string
-	numReplicas int
-	hashKey     HashFunction
-	mu          sync.RWMutex
-	replicaMu   sync.RWMutex
+	ring      map[int]uint8
+	keys      []int
+	items     []string
+	numVnodes int
+	hashKey   HashFunction
+	mu        sync.RWMutex
 }
 
-func NewConsistentHash(hashFunction HashFunction, numReplicas int) *ConsistentHash {
+// NewConsistentHash returns a new consistent hash ring.
+//
+// The given hash function is used to map vnodes and keys to positions on the
+// ring. Ideally, the hash function produces random-looking outputs for distinct
+// inputs.
+//
+// numVnodes decides how many copies of each server replica to place on the
```

Comment:
prolly should just call this arg `vnodes`

Reply:
Done